### DISCUSSION TITLE
how to close a feature request

### DISCUSSION THREAD

**Discussion by dingmeng-xue** [community/community#3546] [Discussions] [answered]
HI Team,

We added feature request in discussions tab. Instead of deleting item, is it possible to close this request? I'm wondering if repo maintainer can close the request.

Thanks,
Dingmeng

**Comment by PranamBhat** [comment]
Hi @dingmeng-xue 

You can not "**Close**" the request. Meantime, you can mark this as "**Answered**" and leave the discussion.

**Reply by dingmeng-xue** [reply]
I didn't figure out how to "Answered" it. https://github.com/Azure/azure-powershell/discussions/14378 

I also cannot see "Close". What I can see is "delete discussion".

**Reply by PranamBhat** [reply]
Go to particular comment on your "Discussion" post. On the top right corner of the "Comment" you will see 3 dots (...).

Click on it. Now click on "**Mark as Answer**".

**Comment by dingmeng-xue** [comment]
Hello, I didn't find butto to "answered" it. could you provide a snapshot?

**Reply by PranamBhat** [reply]
Click on right mark to select "Mark as Answer".

![image](https://user-images.githubusercontent.com/32120033/112849729-f4357300-90c6-11eb-9914-f2b200cc4488.png)

**Reply by dingmeng-xue** [reply]
Do you mean the check mark in the circle? I cannot see that. 

![image](https://user-images.githubusercontent.com/56333063/112851135-44b7cb00-90dd-11eb-8726-537022bad344.png)

**Reply by TechStudent10** [reply]
> Do you mean the check mark in the circle? I cannot see that.

@dingmeng-xue huh, that's weird...

**Reply by dingmeng-xue** [reply]
Is it due to that I'm not author of that item? I'm repo admin and I hope I can close any items instead of contacting authors.

**Comment by PranamBhat** [comment]
Hi @dingmeng-xue 

Where you are checking? Are you using desktop/mobile?

Just refresh your page (**CTRL+F5**) and come to this particular comment, click on "**right mark**" at the top right corner of this comment.

It will select this comment as "**Mark as Answer**".


Thank you!

**Comment by PranamBhat** [comment, marked as answer]
Also, @dingmeng-xue  if you are not the author of that item and you are the repo admin then you can close this item instead of contacting particular author.

Repo owners can mark Discussions as answers as well as triagers, maintainers and the author of a post.


Check this out : [https://docs.github.com/en/discussions/managing-discussions-for-your-community/managing-categories-for-discussions-in-your-repository#default-categories](https://docs.github.com/en/discussions/managing-discussions-for-your-community/managing-categories-for-discussions-in-your-repository#default-categories)

**Reply by dingmeng-xue** [reply]
Got it. If category is not Question/Answer, it cannot be answered. I just changed the category. Thanks.
